documentation/index.html.haml

```diff
@@ -24,6 +24,14 @@ description: List of Narayana documents available for download
         %a{:href => relative("/") + site.current_narayana_api.view}
           %i.icon-external-link
           Narayana Javadocs
+      %p
+        %a{:href=> relative("/") + site.current_narayana_api.viewblacktie}
```

Comment:
Not sure it's correct to have no space before = and space after. Although I'm not 100% sure which one is correct. Having spaces around or not having them at all.